core/CliMulti/Process.php

```diff
@@ -236,4 +240,18 @@ private static function isProcFSMounted()
         return strpos($type, 'proc') === 0;
     }
 
+    /**
+     * @return int[] The ids of the currently running processes
+     */
+     static function getRunningProcesses()
+    {
+        $ids = explode("\n", trim(`ps ex 2>/dev/null | awk '{print $1}'`));
```

Comment:
I did and it showed a message, so I redirected the error output to /dev/null. Now I realize though that awk actually gets executed even if the return code of the first command isn't 0, so I'll add the same redirection too.